src/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -108,17 +108,6 @@ void satcheck_minisat2_baset::lcnf(const bvt &bv)
   clause_counter++;
 }
 
-#ifndef _WIN32
-
-static Minisat::Solver *solver_to_interrupt=nullptr;
-
-static void interrupt_solver(int signum)
-{
-  solver_to_interrupt->interrupt();
-}
-
-#endif
-
```

Comment:
This has only been added as recently as 0ee49334058 - maybe just revert the entire commit (assuming @smowton approves)?

Reply:
It might be a part of an experiment to stop perl script with CTRL+C (which only had to be implemented in the perl script).

Reply:
No, it was intended to facilitate input synthesis, the idea being if you're using a sub-solver to synthesise inputs then you should only give it a little time, since it's not worth losing your solution because of an input synthesis timeout. However there turned out to be too many sources of possible timeouts in CBMC to be practical, so I concluded we should do this by splitting the test-generation process into multiple processes and use process granularity crash/timeout isolation instead.

Long story short, revert that commit at your leisure.